setup.py

```diff
@@ -39,7 +39,7 @@
     ext_modules=[
         Extension("intbitset",
                   ["src/intbitset.c", "src/intbitset_impl.c"],
-                  extra_compile_args=['-O3', '-march=native']
```

Comment:
How would you pich a `-march` that would compile 32 bit on 32 bit platforms and 64 on 64 bits.